spec/unit/recipes/default_spec.rb

```diff
@@ -0,0 +1,20 @@
+#
+# Cookbook Name:: centreon-engine
+# Spec:: default
+#
+# Copyright (c) 2016 The Authors, All Rights Reserved.
+
+require 'spec_helper'
+
+describe 'centreon-engine::default' do
+  context 'When all attributes are default, on an unspecified platform' do
```

Comment:
I think you should update the code generator because we don't do this anymore (testing "unspecified" platform)